cmd/meroxa/turbine/python/cli.go

```diff
@@ -3,14 +3,24 @@ package turbinepy
 import (
 	"github.com/meroxa/cli/cmd/meroxa/turbine"
 	"github.com/meroxa/cli/log"
+	"github.com/meroxa/turbine-core/pkg/server"
 )
 
 type turbinePyCLI struct {
-	logger  log.Logger
-	appPath string
-	tmpPath string
+	logger            log.Logger
+	appPath           string
+	tmpPath           string
+	grpcListenAddress string
+	runner            server.Server
+	builder           server.Server
 }
 
 func New(l log.Logger, appPath string) turbine.CLI {
-	return &turbinePyCLI{logger: l, appPath: appPath}
+	return &turbinePyCLI{
+		logger:            l,
+		appPath:           appPath,
+		grpcListenAddress: server.ListenAddress,
```

Comment:
You might need to rebase your PR off of Lyu's run, so you can include things like:
```suggestion
		grpcListenAddress: internal.RandomLocalAddr(),,
```

Reply:
You need these, otherwise all the runners will be clashing

Reply:
@dianadoherty there is an extra `,`

Reply:
@lyuboxa Just to clarify, do you mean we should be using server.ListenAddress for grpcListenAddress? Or is internal.RandomLocalAddr() ok to use for it?